config/locales/fr.yml

```diff
@@ -10,12 +13,24 @@ fr:
         email: Email
         password: Mot de passe
         remember_me: Se souvenir de moi
+      daily_question:
+        scheduled_at: Date
     errors:
       models:
         user:
           attributes:
             email:
               blank: ne peut être vide
+    errors:
```

Comment:
Duplicated keys here.
You can merge the activerecord.errors keys inside the same tree. ;)